node18/README.md

```diff
@@ -0,0 +1,59 @@
+## Node.js toolbox
+
+### Info
+This is an image of Node.js 16.x version  in toolbox format
+
+### Global npm packages
+This image has pre-installed npm global packages
+
+ - yarn
+ - gulp-cli
+ - grunt-cli
+ - node-sass
+ - bower
+ - typescript
+ - tslint
+
+### Configuration
+Available binary paths for export:
+
+- /usr/bin/node
+- /usr/bin/npm
+- /usr/bin/yarn
+- /usr/bin/gulp
+- /usr/bin/grunt
+- /usr/bin/node-sass
```

Comment:
Is it available, or changed to /usr/bin/sass ?

Reply:
Updated